Added Documentation for asciiview.md needed for advanced hw5

patelpri opened the pull request:
# Advanced HW 5 submission

Closes #136 

Name: Priyank
Uniqname: patelpri

## Description of Changes
added documentation for asciiview command along with examples.

stevietriesenberg commented:
Hey Priyank,

A couple things:
1. Can you remove the "Break it down" headers?  They're a bit redundant.
2. What's the difference between asciiview and aview?  Explain this in detail.
3. From looking at the asciiview webpage, it looks like asciiview has several options.  Can you give some examples of options and what they do?
4. Asciiview has specific controls.  Can you add some brief documentation on these?
5. If you have commands as examples, they should be in the correct code block format, similar to your first example (i.e. "asciiview earth_and_moon.jpg" should be formatted the same as "asciiview [options] filename.jpg" on your page)

patelpri commented:
Closes #140 
Name: Priyank Patel
uniqname: patelpri

Description of all changes
- addressed all the issues mentioned in the comment above. Removed "Break it down" headers, explained the difference between asciiview and aview, showed examples and explained several options and controls, and fixed proper bash command in examples.

stevietriesenberg commented:
Credit awarded, +2 ec
